src/core/js/models/itemsQuestionModel.js

```diff
@@ -97,10 +109,12 @@ export default class ItemsQuestionModel extends BlendedItemsComponentQuestionMod
 
     this.set(props);
 
+    const hasAtLeastSelectableCorrectAnswers = (props._numberOfCorrectAnswers >= selectable);
     const hasRightNumberOfCorrectAnswers = (props._numberOfCorrectAnswers === props._numberOfRequiredAnswers);
     const hasNoIncorrectAnswers = !props._numberOfIncorrectAnswers;
+    const hasNoPartlyCorrectAnswers = !props._numberOfPartlyCorrectAnswers;
 
-    return hasRightNumberOfCorrectAnswers && hasNoIncorrectAnswers;
+    return (hasRightNumberOfCorrectAnswers || hasAtLeastSelectableCorrectAnswers) && hasNoIncorrectAnswers && hasNoPartlyCorrectAnswers;
```

Comment:
suggestion: can we switch these `hasRightNumberOfCorrectAnswers || hasAtLeastSelectableCorrectAnswers` so they are in the same order as the order the variables are assigned. 
`return (hasAtLeastSelectableCorrectAnswers || hasRightNumberOfCorrectAnswers) && hasNoIncorrectAnswers && hasNoPartlyCorrectAnswers;`